src/app/app.component.html

```diff
@@ -1,6 +1,8 @@
 <app-header></app-header>
 
-<div class="top-bar"></div>
+<div class="top-bar">
+  <button mat-raised-button type="button" [routerLink]="['/consultants']" routerLinkActive="router-link-active" >Go Home</button>
```

Comment:
This button should be in the app-header based on the designs. The routerLinkActive attribute is the class that should be applied to the button when the routerLink is the currently active route; I don't see a class called router-link-active so this attribute can be removed. The mat-raised-button directive isn't doing anything to style the button either because MatButtonModule isn't imported in the AppModule (if moving to app-header then it'd need to be in the CoreModule).